spec.html

```diff
@@ -39041,6 +38764,14 @@ <h1>Properties of Generator Instances</h1>
               The execution context that is used when executing the code of this generator.
             </td>
           </tr>
+          <tr>
+            <td>
+              [[GeneratorBrand]]
+            </td>
+            <td>
+              A brand used to distinguish different types of built-in generators. The [[GeneratorBrand]] of generators declared by ECMAScript source text is always ~empty~.
```

Comment:
```suggestion
              A brand used to distinguish different kinds of generators. The [[GeneratorBrand]] of generators declared by ECMAScript source text is always ~empty~.
```
- Different generators don't have different *types* as the term is used in the spec.
- The phrase "built-in generators" suggests that the generators themselves are built-in.